index.html

```diff
@@ -175,13 +175,52 @@ <h1>Skills</h1>
                     <li><span>&bull; GitLab</span></li>
                     <li><span>&bull; Terminal</span></li>
                 </ul>
-                <img class="about-img" src="assert/ic_languages_mob.svg" alt="lightbulb icon" />
+                <img class="about-img" src="/assert/ic_languages_mob.svg" alt="lightbulb icon" />
             </div>
+        </div>
+    </section>
+    <footer>
+        <div class="form-section">
+            <section class="section">
+                <h1 class="section-h"> Get started</h1>
+                <p class="section-p"> Lorem ipsum dolor sit amet, consectetur
+                    adipiscing elit, sed do eiusmod tempor incididunt ut
+                    labore et dolore magna.</p>
+            </section>
+            <form action="https://formspree.io/f/xknyprke" method="POST">
+                <ul>
 
+                    <li> <input class=" name-input" name="text" type="text" placeholder="Name">
+                    </li>
+                    <li> <input class="mail-input" name="email" type="email" placeholder="EMAIL"> </li>
+                    <li><textarea class="textarea-input" name="textarea" id="textarea" cols="30"
+                            rows="10"> hello </textarea></li>
 
-            <div></div>
+                </ul>
+                <button type="submit" class="btn profilelin">
+                    <span class="toprofile">Start collaboration</span>
+                </button>
```

Comment:
- [ ] Please fix this weird distance make sure you have a responsive design 

![image](https://user-images.githubusercontent.com/74744787/165407846-57630803-cde8-4b96-8ef8-234363a86c33.png)

Reply:
Fixed good job 👍